poetry.lock

```diff
@@ -155,10 +169,10 @@ description = "The AWS SDK for Python"
 name = "boto3"
 optional = false
 python-versions = "*"
-version = "1.15.6"
+version = "1.14.60"
 
 [package.dependencies]
-botocore = ">=1.18.6,<1.19.0"
+botocore = ">=1.17.60,<1.18.0"
 jmespath = ">=0.7.1,<1.0.0"
 s3transfer = ">=0.3.0,<0.4.0"
```

Comment:
Why do you downgrade some of the versions of the packages?

Reply:
I guess, something with merge goes wrong. Will revert it